.gitlab/build_and_test.sh

```diff
@@ -15,6 +15,7 @@ cd $BUILD_DIR
 
 cmake -C ../host-configs/llnl/$SYS_TYPE/$HOST_CONFIG ../tests/internal
 make -j8
+make style
```

Comment:
> make style

Does this do anything?

Is the intention that the CI should fail if the formatting is off?
I don't think that will happen by just calling `make style`.